local-running/compose.yaml

```diff
@@ -12,21 +12,22 @@ services:
       IPV_ENDPOINT: http://localhost:4501/
       ORCHESTRATOR_CLIENT_ID: orchestrator # pragma: allowlist secret
       ORCHESTRATOR_CLIENT_JWT_TTL: 900
-      ORCHESTRATOR_CLIENT_SIGNING_KEY: MIGHAgEAMBMGByqGSM49AgEGCCqGSM49AwEHBG0wawIBAQQgOXt0P05ZsQcK7eYusgIPsqZdaBCIJiW4imwUtnaAthWhRANCAAQT1nO46ipxVTilUH2umZPN7OPI49GU6Y8YkcqLxFKUgypUzGbYR2VJGM+QJXk0PI339EyYkt6tjgfS+RcOMQNO # pragma: allowlist secret
-      ORCHESTRATOR_DEFAULT_JAR_ENCRYPTION_PUBLIC_KEY: eyJrdHkiOiAiUlNBIiwibiI6ICIwNDY1cUp3bzhuQ2tDMnR2VjRuaXVXRjZJTTZwTmptZVlzemhUd0hQWTYwOS1IVkF0TzhQb1JMVXlBODZyelEtUXpiVDdYeGJ6Q2pmeVJYb1JGT0dsZVpxVHV3bGMyNWV6RHhWNThiaGVjUGlXRk1hRllPUzFXN3pJRHNWRm8zN2dqanZ0a2NENk9xSzhQS0F2Nm41dFVwaGpEQ2Nubm1wVE1JeUdBbnptUUNiU2tKV3U2Vl9nYzN0aXJBdWdYb1p1a01Db2h4dzNfLWM2cHJoTU4wc21ETnYwcVdtdmEzb3Fva2FiZVB3ZTFPUzcyRFh5WFItVFBkX0R0ejQtdFJyOWp2WndIdWxYNFpjczFCQmJqQnBJaW0zV05ZOGFzdjl5amxCeGtkdC1uY2toQ01aZWtQdVQ3eFdTVHJ2Y2NCX2ZublNVZ0VRV181aXJMTmRucjVNV1EiLCJlIjogIkFRQUIiLCJhbGciOiAiUlMyNTYifQ== # pragma: allowlist secret
+      ORCHESTRATOR_SIGNING_JWK: '{"kty":"EC","kid":"orch-signing-local-running-FI4xysvMVdRtkt6xmO5gqcaTF4Tf9NKD1zdg3T8y69M","use":"sig","d":"OXt0P05ZsQcK7eYusgIPsqZdaBCIJiW4imwUtnaAthU","crv":"P-256","x":"E9ZzuOoqcVU4pVB9rpmTzezjyOPRlOmPGJHKi8RSlIM","y":"KlTMZthHZUkYz5AleTQ8jff0TJiS3q2OB9L5Fw4xA04"}' # pragma: allowlist secret
+      AUTH_SIGNING_JWK: '{"kty":"EC","kid":"auth-signing-local-running-FI4xysvMVdRtkt6xmO5gqcaTF4Tf9NKD1zdg3T8y69M","use":"sig","d":"OXt0P05ZsQcK7eYusgIPsqZdaBCIJiW4imwUtnaAthU","crv":"P-256","x":"E9ZzuOoqcVU4pVB9rpmTzezjyOPRlOmPGJHKi8RSlIM","y":"KlTMZthHZUkYz5AleTQ8jff0TJiS3q2OB9L5Fw4xA04"}' # pragma: allowlist secret
+      ORCHESTRATOR_DEFAULT_JAR_ENCRYPTION_PUBLIC_JWK: '{"kty":"RSA","kid":"test-encryption-key","use":"enc","e":"AQAB","alg":"RS256","n":"0465qJwo8nCkC2tvV4niuWF6IM6pNjmeYszhTwHPY609-HVAtO8PoRLUyA86rzQ-QzbT7XxbzCjfyRXoRFOGleZqTuwlc25ezDxV58bhecPiWFMaFYOS1W7zIDsVFo37gjjvtkcD6OqK8PKAv6n5tUphjDCcnnmpTMIyGAnzmQCbSkJWu6V_gc3tirAugXoZukMCohxw3_-c6prhMN0smDNv0qWmva3oqokabePwe1OS72DXyXR-TPd_Dtz4-tRr9jvZwHulX4Zcs1BBbjBpIim3WNY8asv9yjlBxkdt-nckhCMZekPuT7xWSTrvccB_fnnSUgEQW_5irLNdnr5MWQ"}'
       ORCHESTRATOR_PORT: 4500
       ORCHESTRATOR_REDIRECT_URL: http://localhost:4500/callback
       EVCS_ACCESS_TOKEN_ENDPOINT: https://mock.credential-store.build.account.gov.uk/generate
       EVCS_ACCESS_TOKEN_TTL: 60
-      EVCS_ACCESS_TOKEN_SIGNING_KEY_JWK: '{"kty":"EC","d":"OXt0P05ZsQcK7eYusgIPsqZdaBCIJiW4imwUtnaAthU","crv":"P-256","x":"E9ZzuOoqcVU4pVB9rpmTzezjyOPRlOmPGJHKi8RSlIM","y":"KlTMZthHZUkYz5AleTQ8jff0TJiS3q2OB9L5Fw4xA04"}' # pragma: allowlist secret
+      EVCS_ACCESS_TOKEN_SIGNING_JWK: '{"kty":"EC","kid":"evcs-token-local-running-FI4xysvMVdRtkt6xmO5gqcaTF4Tf9NKD1zdg3T8y69M","use":"sig","d":"OXt0P05ZsQcK7eYusgIPsqZdaBCIJiW4imwUtnaAthU","crv":"P-256","x":"E9ZzuOoqcVU4pVB9rpmTzezjyOPRlOmPGJHKi8RSlIM","y":"KlTMZthHZUkYz5AleTQ8jff0TJiS3q2OB9L5Fw4xA04"}' # pragma: allowlist secret
```

Comment:
I don't think this is actually specific to local-running - perhaps it should be, but at the moment I think EVCS stub just accepts a single signing key?

Reply:
This is another place that we use the same test key - so the `evcs-token-local-running` part is just to differentiate it. But yeah, the stub will be set up to just use that key